cmd/gotk/get_helmrelease.go

```diff
@@ -62,26 +73,52 @@ func getHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	table := tablewriter.NewWriter(os.Stdout)
+	tableRowHeaders := []string{"Name", "Revision", "Ready", "Message", "Suspended"}
+	if allNamespaces {
+		tableRowHeaders = append(tableRowHeaders, "Namespace")
+	}
+	table.SetHeader(tableRowHeaders)
+	table.SetRowLine(true)
+
 	for _, helmRelease := range list.Items {
+		row := []string{}
 		if helmRelease.Spec.Suspend {
-			logger.Successf("%s is suspended", helmRelease.GetName())
+			row = []string{
+				helmRelease.GetName(),
+				helmRelease.Status.LastAppliedRevision,
+				"false",
+				"suspended",
+				strings.Title(strconv.FormatBool(helmRelease.Spec.Suspend)),
+			}
+			if allNamespaces {
+				row = append(row, helmRelease.Namespace)
+			}
+			table.Append(row)
 			continue
 		}
-		isInitialized := false
 		if c := meta.GetCondition(helmRelease.Status.Conditions, meta.ReadyCondition); c != nil {
-			switch c.Status {
-			case corev1.ConditionTrue:
-				logger.Successf("%s last applied revision %s", helmRelease.GetName(), helmRelease.Status.LastAppliedRevision)
-			case corev1.ConditionUnknown:
-				logger.Successf("%s reconciling", helmRelease.GetName())
-			default:
-				logger.Failuref("%s %s", helmRelease.GetName(), c.Message)
+			row = []string{
+				helmRelease.GetName(),
+				helmRelease.Status.LastAppliedRevision,
+				string(c.Status),
+				c.Message,
+				strings.Title(strconv.FormatBool(helmRelease.Spec.Suspend)),
+			}
+		} else {
+			row = []string{
+				helmRelease.GetName(),
+				helmRelease.Status.LastAppliedRevision,
+				string(corev1.ConditionFalse),
+				"waiting to be reconciled",
+				strings.Title(strconv.FormatBool(helmRelease.Spec.Suspend)),
 			}
-			isInitialized = true
 		}
```

Comment:
This can theoretically be changed to:

```suggestion
		row := []string{
			helmRelease.GetName(),
			helmRelease.Status.LastAppliedRevision,
			string(corev1.ConditionFalse),
			"waiting to be reconciled",
			strings.Title(strconv.FormatBool(helmRelease.Spec.Suspend)),
		}
		if c := meta.GetCondition(helmRelease.Status.Conditions, meta.ReadyCondition); c != nil {
			row = []string{
				helmRelease.GetName(),
				helmRelease.Status.LastAppliedRevision,
				string(c.Status),
				c.Message,
				strings.Title(strconv.FormatBool(helmRelease.Spec.Suspend)),
			}
		}
```